src/components/App/AppSagas.js

```diff
@@ -11,11 +11,13 @@ import {
   CHANGE_PAGE_ACTION,
 } from 'components/App/AppActions';
 import { AUTH_SUCCESS_ACTION } from 'components/LoginPage/LoginPagesActions.js';
-import { PAGES_ENUM } from 'components/App/AppConstants';
+// import { PAGES_ENUM } from 'components/App/AppConstants';
```

Comment:
Do we still need `PAGES_ENUM` if we're using react router? Let's delete the enum?

Reply:
Ok. I leave the comments so it's immediately seen where code changes was made and when needed I can easily uncomment them to restore app working state, so please don't concentrate on deleting commented code, all comments will be deleted once all file/component are refactored. I regularly parsing through files so they will not be left undeleted